deploy.sh

```diff
@@ -0,0 +1,22 @@
+GOOGLE_PROJECT=$1
+
+#jenkins needs access to the projects so it can deploy
+# command TBD (that's either already the case or we use some credential here to make it so)
+
+#render the endpoints json
+python lib/endpoints/endpointscfg.py get_openapi_spec main.BondApi --hostname $GOOGLE_PROJECT.appspot.com
+
+#deploy google endpoints
+gcloud -q endpoints services deploy linkv1openapi.json
+
+#SERVICE_VERSION in app.yaml needs to match this
+SERVICE_VERSION=`gcloud endpoints services describe $GOOGLE_PROJECT.appspot.com --format=json | jq .serviceConfig.id`
+
+#render app.yaml and config.ini for environment with SERVICE_VERSION and GOOGLE_PROJECT
+docker run -v $PWD:/app -e GOOGLE_PROJ=$GOOGLE_PROJECT -e SERVICE_VERSION=$SERVICE_VERSION -e INPUT_PATH=/app -e OUT_PATH=/app/output render-templates.sh
+
+#install dependencies to lib folder
+pip install -t lib -r requirements.txt
+
+#deploy the app to the specified project
+gcloud -q app deploy app.yaml --project=$GOOGLE_PROJECT
```

Comment:
this will need to be `gcloud -q app deploy output/app.yaml --project=$GOOGLE_PROJECT`, if we are rendering configs to `output` directory

Reply:
I can change as I need to test git push triggers on this repo in jenkins